fhir-server-test/src/test/resources/test.properties

```diff
@@ -39,11 +39,12 @@ fhirclient.http.return.pref = minimal
 test.client.oauth.enabled = false
 test.websocket.enabled = false
 test.kafka.enabled = false
-test.bulkdata.import.enabled = false
-test.bulkdata.export.enabled = false
-test.bulkdata.useminio = false
-test.bulkdata.minio.username = minio
-test.bulkdata.minio.password = change-password
```

Comment:
do we still exercise minio during the CI, or is it only the file-based storage provider that gets tested now?

Reply:
It's  only the file-base provider. i can  enable minio too  - I'd just have to include a different set of tests. 
We could do that with a separate issue.  Let me  know and I can create it.

Reply:
I think it would be nice to exercise both providers (since we already did the work to enable minio).  Agree it can be a future thing.

Reply:
both providers are in the IT tests